docs/visualization/matplotlib_integration.rst

```diff
@@ -40,6 +40,33 @@ though the second line is given in ``cm``:
 
     ax.plot([1, 2, 3] * u.cm)
 
+If you want more control, you can explicitly set the units for the x-axis
+and/or y-axis by passing a unit argument to ``ax.xaxis.set_units()`` or
+``ax.yaxis.set_units()``.
+
+For example, you can set the y-axis units to centimeters:
```

Comment:
On the one hand, I like the complete example, on the other there is an advantage to brevity. So I leave this up to you, but I think it would also work to just continue with the above example, i.e.,

```
... plot::
    :include-source:
    :context:

    ax.yaxis.set_units(u.mm)

This takes precedence over the unit used, regardless of the data's units or plot order.
You can find ...
```

One small advantage of this is that it shows one can set the unit also after the lines were added (I checked that worked).